src/index.js

```diff
@@ -0,0 +1,60 @@
+import './styles/style.css';
+import Alltask from './add.js';
+import { real, displayUI, newTask } from './reload.js';
+import { read, addButton } from './addtask.js';
+
+let tasks = [];
+addButton.addEventListener('click', (e) => {
+  tasks = JSON.parse(localStorage.getItem('list')) || [];
+  e.preventDefault();
+  const findIndex = tasks.length;
+  const task1 = new Alltask(read.value, false, findIndex + 1);
+  tasks.push(task1);
+  real(task1);
+  read.value = '';
+  localStorage.setItem('list', JSON.stringify(tasks));
+});
+
+newTask.addEventListener('click', (e) => {
+  tasks = JSON.parse(localStorage.getItem('list')) || [];
+
+  if (e.target.classList.contains('remove')) {
+    e.target.parentElement.parentElement.remove();
+    const k = parseInt(e.target.id, 10);
+    const newArray = tasks.filter((Objects) => Objects.index !== k);
+    for (let i = 0; i < newArray.length; i += 1) {
+      newArray[i].index = 1;
+      newArray[i].index += i;
+    }
+    localStorage.setItem('list', JSON.stringify(newArray));
+    window.location.reload();
+  } else if (e.target.checked) {
+    const k = parseInt(e.target.id, 10);
+    tasks[k - 1].completed = true;
+    localStorage.setItem('list', JSON.stringify(tasks));
+  } else {
+    const k = parseInt(e.target.id, 10);
```

Comment:
DRY principle: I would suggest putting those lines into a separate function, because `else if` and `else` statements are repeated. 

Please checkout gist I created with index.js code that I updated:
https://gist.github.com/whid-bb/fc3932bee5c26c4db5501e1f4cfc5348